src/SignalR/clients/csharp/Client/test/FunctionalTests/Hubs.cs

```diff
@@ -279,4 +279,14 @@ public class HubWithAuthorization2 : Hub
     {
         public string Echo(string message) => TestHubMethodsImpl.Echo(message);
     }
+
+    public class NegotiateVersion12Hub : Hub
```

Comment:
Don't need these hubs really

Reply:
Yeah, I guess we can just reuse other ones and ax these